lessons/Early-Math/equivalence/langs/en-us/pages/true-or-false.adoc

```diff
@@ -0,0 +1,22 @@
+= True or False?
+
+++++
+<style>
+div.circleevalsexp { width: auto; }
+td > .content > .paragraph > * { vertical-align: middle; }
+</style>
+++++
+
+Is the equation represented by the two Circles of Evaluation true or false? Explain why, and then describe what you would change to make the _false_ equations _true_.
```

Comment:
I had to look at the solution page for this to know what you thought would change to make false equations true...
Since changing the operations would make the problem pretty unrelated to what I'm looking at, that wasn't intuitive for me.

wondering if you'd feel comfortable changing the prompt to something more open-ended like:

Is the equation represented by the two Circles of Evaluation true or false? Explain why or why not.

Reply:
yeah. This is a good suggestion. Gonna have to change directions / answers on all true/false worksheets.